src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs

```diff
@@ -408,7 +447,7 @@ private async void FileList_PreviewKeyDown(object sender, KeyRoutedEventArgs e)
 			}
 			else if (e.Key == VirtualKey.Right) // Right arrow: switch focus to next column
 			{
-				if (IsRenamingItem || (ParentShellPageInstance is not null && ParentShellPageInstance.ToolbarViewModel.IsEditModeEnabled)) 
+				if (IsRenamingItem || (ParentShellPageInstance is not null && ParentShellPageInstance.ToolbarViewModel.IsEditModeEnabled))
```

Comment:
Identifical to l.441, could be merged inside a function.

Reply:
I think that would be a change more suitable for another PR.